features/importing/archivist.feature

```diff
@@ -81,8 +84,11 @@ Feature: Archivist bulk imports
       | user1 | a@ao3.org |
     When I go to the import page
       And I import the work "http://ao3testing.dreamwidth.org/593.html" by "name1" with email "a@ao3.org" and by "name2" with email "b@ao3.org"
-    Then 1 email should be delivered to "a@ao3.org"
+    When the system processes jobs
+      Then 1 email should be delivered to "a@ao3.org"
```

Comment:
This step is indented two spaces too far.

Also, I think we need to update [the specs for this email](https://github.com/otwcode/otwarchive/blob/17307a8a6294d2af7fab882ea5aa14eb786d51af/spec/mailers/user_mailer_spec.rb#L4-L62) to include `it_behaves_like "a translated email"`, like we've done with the other translated emails.